docs/execution-guide.md

```diff
@@ -32,43 +31,78 @@ PID|1|7777555^4^M11^test^MR^University Hospital^19241011^19241012|PATID7755^5^M1
+     
+     > A second pop up will appear stating "You have no storage mounted". Click `Create storage`.![azure-cloud-create-storage](./images/azure-cloud-create-storage.png)
+1. Confirm that you're in Bash mode by checking that the dropdown in the top left under the `Microsoft Azure` header has `Bash` selected.![azure-cloud-shell](./images/azure-cloud-shell.png)
+1. Then in the terminal, type the command `az login` and press enter.![azure-cloud-shell-login](./images/azure-cloud-shell-login.png)
+1. Copy the authentication code provided.![azure-cloud-copy-code](./images/azure-cloud-copy-code.png)
+1. Click the device login link, and paste in the authentication code.![azure-device-login](./images/azure-device-login.png)
+1. Follow the prompts to complete login. After logging in, you should see a pop up that says "You have signed in...".![azure-cloud-logged-in](./images/azure-cloud-logged-in.png)
+1. Return to the tab with Azure Cloud Shell open.![azure-cloud-logged-in-terminal](./images/azure-cloud-logged-in-terminal.png)
+
+Now that we're authenticated, we're going to try to search for a patient named "John Doe" in the VXU message we ran earlier. 
+1. A couple of the commands below will require you to update the URL in the command with the name of your FHIR server. To get the name of your FHIR server, go back to the tab with [portal.azure.com](portal.azure.com) open. Then in the search bar, type in `Azure API for FHIR` and select this option in the search dropdown.![azure-fhir-api-search](./images/azure-fhir-api-search.png)
+1. You should see your FHIR server which will have the following form: `{environment}fhir{client-id}`. Click into your FHIR server.![azure-find-fhir-server](./images/azure-find-fhir-server.png)
+1. Copy the name of your FHIR server which will have the following form: (`{environment}fhir{client-id}`).![azure-fhir-server](./images/azure-fhir-server.png)
+1. Go back to the tab with Cloud Shell open. Copy and paste the command below, then replace the **FHIR_SERVER_NAME** text with the pasted name of your FHIR server. This will save the name of your FHIR server into a variable that will be referenced in later commands. <pre>export FHIR_SERVER=<b>FHIR_SERVER_NAME</b></pre>![azure-cloud_set_FHIR_variable](./images/azure-cloud_set_FHIR_variable.png)
+1. Copy and paste the below command into the terminal and hit enter. This gets you a bearer token that will be used to authenticate in the next command:<pre>
+token=$(az account get-access-token --resource=https://$FHIR_SERVER.azurehealthcareapis.com --query accessToken --output tsv)</pre>
+1. Copy and paste the below command into the terminal and hit enter. This uses the bearer token from above to authenticate and search for the "John Doe" user: <pre>RESPONSE=$(curl -X GET --header "Authorization: Bearer $token" "https://$FHIR_SERVER.azurehealthcareapis.com/Patient?family=DOE&given=JOHN")</pre>
```

Comment:
Could we link to our sample FHIR API queries documentation in this section? https://github.com/CDCgov/phdi-azure/blob/main/docs/basic-guide-to-fhir-api-calls.md

Reply:
added link!